sktime/forecasting/regularized_var.py

```diff
@@ -0,0 +1,224 @@
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements Regularized (L1/L2) VAR using cvxpy."""
+
+# __author__ = [meraldoantonio]
+
+import numpy as np
+import pandas as pd
+
+from sktime.forecasting.base import BaseForecaster
+
+
+class RegularizedVAR(BaseForecaster):
+    """Custom forecaster using VAR with regularization.
+
+    Parameters
+    ----------
+    lags : int
+        The number of lagged values to include in the model.
+    L1_penalty : float, optional (default=0.0)
+        The L1 regularization penalty.
+    L2_penalty : float, optional (default=0.0)
+        The L2 regularization penalty.
+    """
+
+    _tags = {
+        "scitype:y": "both",
+        "authors": ["meraldoantonio"],
+        "python_dependencies": "cvxpy",
```

Comment:
the model will not be tested in CI if `cvxpy` is not present in the env - so you need to add it at least in the `all_extras` dep set in `pyproject,toml` to ensure test coverage

Reply:
Noted! I've changed the logic of the forecaster to simply use a user-defined regressor for fitting, so I removed all cvxpy references